subsys/bluetooth/controller/hal/nrf5/radio.c

```diff
@@ -20,6 +20,8 @@
 #error "Platform not defined."
 #endif
 
+#include "include/hal_radio_nrf5.h"
```

Comment:
dont have include folder, as these new files are private use in hal/radio.c and not exported as interfaces to other modules.

Reply:
This "include" was an easy way to avoid creating a Cmake lists-structure, as I had created an internal folder "include" to put all the headers. Is this a problem/bad practice?